README.md

```diff
@@ -17,7 +17,7 @@ See: https://b3n.org/automatic-ripping-machine
     - Retrieve title from disc or Windows Media MetaServices API to name the folder "movie title (year)" so that Plex or Emby can pick it up
     - Determine if video is Movie or TV using OMDb API
     - Rip using MakeMKV (can rip all features or main feature)
-    - Eject disc and queue up Handbrake transcoding when done 
```

Comment:
Can you clear out these extra whitespaces at the end of the lines? Here and a few other places.